open-policy-agent/authorization/serviceaccount_automounttoken_prevention/template.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: templates.gatekeeper.sh/v1beta1
+kind: ConstraintTemplate
+metadata:
+  name: k8sserviceaccountautomounttokenprevention
```

Comment:
@tommeramber @michaelkotelnikov 
The is very long, suggest to change to something shorter like 'k8snotokenautomount'